src/example-types/esTwoLevelAggregation.js

```diff
@@ -72,7 +75,12 @@ module.exports = {
                 key: bucket.key || key,
                 doc_count: bucket.doc_count,
               },
-              bucket.twoLevelAgg
+              bucket.twoLevelAgg ||
+                bucket.stats ||
```

Comment:
Having explicit knowledge of `stats` in esTwoLevel aggregation is not ideal. Some options that come to mind:

- We could have an explicit list of "multivalue" metrics (like stats and extended stats) which we know to just flatten onto the bucket (instead of being `stats.min/stats.max/etc` for example)
- We could assume that _all_ metrics that don't have `.value` or `.values` are multivalue metrics

The second option probably seems like it would be the most generic and future proof option - we just flatten when there's not a  `.value` or `.values` prop and replace `F.getOrReturn('value')` with something like `x => F.cascade(['value', 'values'], x, x). I don't think we currently have anything like `F.flattenObjectBy`, but it would be trivial to extend F.flattenObject to optionally take a third param for `isFlatObject`: https://github.com/smartprocure/futil-js/blob/master/src/object.js#L62